src/scoreboard/main.js

```diff
@@ -0,0 +1,28 @@
+// Javascript
+import 'bootstrap/dist/js/bootstrap.min';
+import InitScoreboardController from './controllers/initscoreboardcontroller';
+import InitScoreboardModel from './models/initscoreboardmodel';
+
+// Stylesheets
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './views/css/scoreboard.css';
+
+$(document).ready(function ()
+{
+    window.GAME = {
```

Comment:
You _may_ not want to use `GAME` here.

Sticking to _may_, as it does not matter too much.

Reply:
Yes, saw that too. What do you prefer. Doesn't matter to me...

Reply:
Any comments regarding this?

Reply:
Just go with it. It's not pretty, but I don't think anyone will trip over this, as the scoreboard is well separated from the actual game.